rules/xcodeproj.bzl

```diff
@@ -99,6 +99,12 @@ def _xcodeproj_aspect_impl(target, ctx):
     bazel_bin_subdir = "%s/%s" % (target.label.workspace_root, target.label.package)
 
     if AppleBundleInfo in target:
+        swift_objc_header_path = None
+        if SwiftInfo in target:
+            for h in target[apple_common.Objc].header.to_list():
```

Comment:
could this iterate over `direct_headers` to limit the amount of depset flattening that needs to be done?